### PR TITLE
Introducing ads.txt

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?

Introduces the Guardian `ads.txt` file. See standard for details https://iabtechlab.com/ads-txt/

### CODE REVIEW

File: ads.txt [hunk 1]
```diff
@@ -28,7 +28,7 @@ google.com, pub-2012933198307164, DIRECT
 google.com, ca-pub-7526622147976638, DIRECT
 google.com, ca-pub-4979300716531995, DIRECT
 google.com, pub-5107472231497753, DIRECT
-go.sonobi.com, d089631d2d, DIRECT
+go.sonobi.com, d089631d2d, RESELLER
```
Comment (ScottPainterGNM) [inline ads.txt]: I think we should double check this with Danny and Drew, just in case there's a reason they've put them as DIRECT originally.

Comment (ScottPainterGNM) [reply ads.txt]: Confirmed this needs to be direct, can you change back?

image

**Comment by shtukas** [comment]
This has been business signed off by Andrew H., Tony B. and Brendan C.

**Comment by TBonnin** [comment]
If you want this file to be served you need to add it to `commercial/public` and create a route in commercial `Routes` file (ex: https://github.com/guardian/frontend/blob/94d2e7c69bbde1ea4e507959af3c2d9d66c54915/diagnostics/conf/routes#L9-L9)

**Comment by TBonnin** [comment]
You'll need to add it to the router conf as well https://github.com/guardian/platform/blob/master/router/files/router.conf

**Comment by shtukas** [comment]
Ok, so I will be pushing this live now. Thomas and I will check that it works fine by bypassing the router and then we will modify the platform config.

**Review by TBonnin — APPROVED** [review]
lgtm. 
Do you care if the file is not accessible when running dev-build? If yes you will need to add the route to `dev-build/conf/routes` but that's up to you

**Comment by shtukas** [comment]
Thanks. I will leave it as it is as there is no real need to have it available on local.

**Pull request merged**